dump/dbus_util.cpp

```diff
@@ -160,13 +160,11 @@ bool isHostRunning(sdbusplus::bus::bus& bus)
             return false;
         }
 
-        ProgressStages bootProgess = sdbusplus::xyz::openbmc_project::State::
+        ProgressStages bootProgress = sdbusplus::xyz::openbmc_project::State::
             Boot::server::Progress::convertProgressStagesFromString(*progPtr);
-        if ((bootProgess == ProgressStages::SystemInitComplete) ||
-            (bootProgess == ProgressStages::OSStart) ||
-            (bootProgess == ProgressStages::OSRunning))
+        if (bootProgress == ProgressStages::OSRunning)
```

Comment:
I was wondering that in the meanwhile, is someone also working on returning the error from PHYP in case of failure. We might have addressed this particular scenario, but it might be better to address the root cause

Reply:
Yeah I tried that at present pldm_send used in this repository is an async call so changed to pldm_send_recv to see if we get any error, if so thought of exploring coroutines, but it was not returning any error. It is simply ignoring the command.